### PR TITLE
fix: MD039/no-space-in-links

### PR DESCRIPTION
**Pull request by nschonni**
Spaces inside link

### CODE REVIEW

File: Skype/SfbServer/sfb-server-updates.md [hunk 1]
```diff
@@ -114,9 +114,9 @@ Package Name/Link|Release Date|
 |[Lync Server 2013 Resource Kit Tools](https://www.microsoft.com/en-us/download/details.aspx?id=36821) |January 2014 |
 |[Lync Server 2013 Stress and Performance Tool](https://www.microsoft.com/en-us/download/details.aspx?id=36819) |March 2013 |
 |[Lync Server 2013 Whiteboard Archiving Viewer](https://www.microsoft.com/en-us/download/details.aspx?id=36822) |February 2013 |
-|[Lync VDI 2013 plugin (32-bit) ](https://www.microsoft.com/en-us/download/details.aspx?id=35457) |October 2012 |
-|[Lync VDI 2013 plugin (32-bit) ](https://www.microsoft.com/en-us/download/details.aspx?id=35454) |October 2012 |
-|[New Office Visio Stencil ](https://www.microsoft.com/en-us/download/details.aspx?id=35772) |July 2016 |
+|[Lync VDI 2013 plugin (32-bit)](https://www.microsoft.com/en-us/download/details.aspx?id=35457) |October 2012 |
+|[Lync VDI 2013 plugin (64-bit)](https://www.microsoft.com/en-us/download/details.aspx?id=35454) |October 2012 |
```
Comment (nschonni) [inline Skype/SfbServer/sfb-server-updates.md]: This is the only non-whitespace change. Noticed the duplicate text, and confirmed by going to the download page that one was for the 64-bit

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/deploy/deploy-enterprise-voice/deploy-network.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ description: "Create or modify network regions, network sites, and associate net
...
+The advanced Enterprise Voice features are [call admission control](../../plan-your-deployment/enterprise-voice-solution/call-admission-control.md), [media bypass](../../plan-your-deployment/enterprise-voice-solution/media-bypass.md), [location-based routing](../../plan-your-deployment/enterprise-voice-solution/location-based-routing.md), and [E9-1-1](../../plan-your-deployment/enterprise-voice-solution/emergency-services.md). These features all require you to create network regions, network sites, and subnets. For example, all of these features require that each subnet in your topology be associated with a specific network site, and each network site must be associated with a network region. For more information on these terms, see [Network settings for the advanced Enterprise Voice features in Skype for Business Server](../../plan-your-deployment/enterprise-voice-solution/network-settings-for-advanced-features.md)
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/deploy-enterprise-voice/deploy-network.md]:
```suggestion
The advanced Enterprise Voice features are [call admission control](../../plan-your-deployment/enterprise-voice-solution/call-admission-control.md), [media bypass](../../plan-your-deployment/enterprise-voice-solution/media-bypass.md), [location-based routing](../../plan-your-deployment/enterprise-voice-solution/location-based-routing.md), and [E9-1-1](../../plan-your-deployment/enterprise-voice-solution/emergency-services.md). These features all require you to create network regions, network sites, and subnets. For example, all of these features require that each subnet in your topology be associated with a specific network site, and each network site must be associated with a network region. For more information on these terms, see [Network settings for the advanced Enterprise Voice features in Skype for Business Server](../../plan-your-deployment/enterprise-voice-solution/network-settings-for-advanced-features.md).
```

File: Skype/SfbServer/management-tools/use-scom-management-pack/watcher-nodes.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Watcher nodes are computers that periodically run Skype for Business Server synt
...
+- **Non-default** Tests that watcher nodes do not run by default. (For details, see the description of the Default type.) However, the watcher node can be enabled to run any of the Non-default synthetic transactions. You can do this when you create the watcher node (by using the New-CsWatcherNodeConfiguration cmdlet), or any time after the watcher node has been created. Note that many of the Non-default synthetic transactions require extra setup steps. For details about these steps, see Special Setup Instructions for Synthetic Transactions. For more details about these steps, see [Special Setup Instructions for Synthetic Transactions](test-users-and-settings.md#special_synthetictrans).
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/management-tools/use-scom-management-pack/watcher-nodes.md]:
```suggestion
- **Non-default** Tests that watcher nodes do not run by default. (For details, see the description of the Default type.) However, the watcher node can be enabled to run any of the Non-default synthetic transactions. You can do this when you create the watcher node (by using the New-CsWatcherNodeConfiguration cmdlet), or any time after the watcher node has been created. Note that many of the Non-default synthetic transactions require extra setup steps. For more details about these steps, see [Special Setup Instructions for Synthetic Transactions](test-users-and-settings.md#special_synthetictrans).
```

File: Skype/SfbServer/plan-your-deployment/upgrade.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ Make a plan that includes:
 1. An understanding of your current topology.
     
     > [!NOTE]
-    > Be sure to uninstall LRS Admin tool for Lync Server 2013 before running In-Place Upgrade. The LRS Admin Tool for Lync Server 2013 cannot coexist with Skype for Business Server 2015. After running In-Place Upgrade install the new LRS Admin tool, see [Microsoft Lync Room System Administrative Web Portal for Skype for Business Server 2015 ](https://go.microsoft.com/fwlink/?LinkID=544807)
+    > Be sure to uninstall LRS Admin tool for Lync Server 2013 before running In-Place Upgrade. The LRS Admin Tool for Lync Server 2013 cannot coexist with Skype for Business Server 2015. After running In-Place Upgrade install the new LRS Admin tool, see [Microsoft Lync Room System Administrative Web Portal for Skype for Business Server 2015](https://go.microsoft.com/fwlink/?LinkID=544807)
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/plan-your-deployment/upgrade.md]:
```suggestion
    > Be sure to uninstall LRS Admin tool for Lync Server 2013 before running In-Place Upgrade. The LRS Admin Tool for Lync Server 2013 cannot coexist with Skype for Business Server 2015. After running In-Place Upgrade install the new LRS Admin tool. See [Microsoft Lync Room System Administrative Web Portal for Skype for Business Server 2015](https://go.microsoft.com/fwlink/?LinkID=544807) for more details.
```

File: Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/plan-phone-system-with-on-premises-pstn-connectivity.md [hunk 5]
```diff
@@ -103,9 +103,9 @@ In addition you must ensure the following:
...
 
-    If you are using Lync Server 2013, see [ Lync Server 2013 hybrid](https://technet.microsoft.com/EN-US/library/jj204805%28v=ocs.15%29.aspx).
+    If you are using Lync Server 2013, see [Lync Server 2013 hybrid](https://technet.microsoft.com/EN-US/library/jj204805%28v=ocs.15%29.aspx).
 
-- **(Recommended) Active Directory Federation Services (AD FS)** We recommend deploying AD FS to support Single Sign-on. For more information, see [ Active Directory Federation Services (AD FS)](https://technet.microsoft.com/en-us/library/cc736690%28v=ws.10%29.aspx).
+- **(Recommended) Active Directory Federation Services (AD FS)** We recommend deploying AD FS to support Single Sign-on. For more information, see [Active Directory Federation Services (AD FS)](https://technet.microsoft.com/en-us/library/cc736690%28v=ws.10%29.aspx).
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/plan-phone-system-with-on-premises-pstn-connectivity.md]:
```suggestion
- **(Recommended) Active Directory Federation Services (AD FS).** We recommend deploying AD FS to support Single Sign-on. For more information, see [Active Directory Federation Services (AD FS)](https://technet.microsoft.com/en-us/library/cc736690%28v=ws.10%29.aspx).
```

File: Teams/view-guests.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ appliesto:
 
 View guest users in a team
 ==========================
-Anyone in a team can see all members of a team, including guests, via **Manage team** in Microsoft Teams. [See who's on a team or in a channel ](https://support.office.com/article/see-who-s-on-a-team-or-in-a-channel-5c6be9be-9c45-4a0f-a1a0-f332b23cb6b7) has more details.
+Anyone in a team can see all members of a team, including guests, via **Manage team** in Microsoft Teams. [See who's on a team or in a channel](https://support.office.com/article/see-who-s-on-a-team-or-in-a-channel-5c6be9be-9c45-4a0f-a1a0-f332b23cb6b7) has more details.
```
Comment (JohanFreelancer9) [inline Teams/view-guests.md]:
```suggestion
Anyone in a team can see all members of a team, including guests, via **Manage team** in Microsoft Teams. Refer to [See who's on a team or in a channel](https://support.office.com/article/see-who-s-on-a-team-or-in-a-channel-5c6be9be-9c45-4a0f-a1a0-f332b23cb6b7) for more details.
```

**Comment by scanum** [comment]
@jambirk  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**